pkgs/development/libraries/libnest2d/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, cmake, clipper, nlopt, boost, python3 }:
+
+stdenv.mkDerivation rec {
+  version = "2020-10-09-unstable";
+  pname = "libnest2d";
+  format = "other";
+
+  # This revision is waiting to be merged upstream
+  # Once it has been merged, this should be switched to it
+  # Upstream PR: https://github.com/tamasmeszaros/libnest2d/pull/18
+  src = fetchFromGitHub {
+    owner = "Ultimaker";
+    repo = "libnest2d";
+    rev = "31391fd173249ad9b906390058e13b09238fadc8";
+    sha256 = "1hzqi4z55x76rss3xk7hfqhy9hcaq2jaav5jqxa1aqmbvarr2gla";
+  };
+
+  buildInputs = [ python3 ];
```

Comment:
```suggestion
```
Python is not required to build libnest2d.

Reply:
Thanks for picking those up @gebner! I've applied those changes.